rti/mosaic-rti-api/src/main/java/org/eclipse/mosaic/rti/api/FederateAmbassador.java

```diff
@@ -41,7 +42,7 @@ public interface FederateAmbassador extends Comparable<FederateAmbassador> {
      * @return the {@link FederateExecutor} which starts the federate
      */
     @Nonnull
-    FederateExecutor createFederateExecutor(String host, int port, OperatingSystem os);
+    FederateExecutor createFederateExecutor(String host, int port, OperatingSystem os) throws FileNotFoundException;
```

Comment:
Since we don't do any special error handling in the case of file not found, I would avoid changing the API method signature and not use checked exception here, instead throwing an `IllegalArgumentException` or `IllegalStateException` in `OmnetppAmbassador`.